casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -13,17 +13,25 @@ import coop.rchain.casper.util.Sorting.byteArrayOrdering
 import coop.rchain.casper.util.rholang.RuntimeManager.StateHash
 import coop.rchain.casper.util.rholang.{InternalProcessedDeploy, RuntimeManager}
 import coop.rchain.casper.util.{BondsParser, VaultParser}
+import coop.rchain.crypto.PublicKey
+import coop.rchain.crypto.codec.Base16
 import coop.rchain.shared.{Log, LogSource, Time}
 
 final case class Genesis(
     shardId: String,
     timestamp: Long,
     proofOfStake: ProofOfStake,
     vaults: Seq[Vault],
-    supply: Long
+    supply: Long,
+    publicKey: PublicKey
```

Comment:
`Secp256k1.newKeyPair._2` doesn't work out since during replay we will get another value. And in general we want to keep things referentially transparent. Instead I changed it to `PublicKey(Array[Byte]())`.